primitives/src/identity_id.rs

```diff
@@ -162,6 +162,16 @@ impl IdentityId {
     }
 }
 
+impl Into<BTreeSet<PortfolioId>> for IdentityId {
+    // Returns a btreeset that contains default portfolio for the identity.
+    // Useful for tests.
```

Comment:
If it is only used in the test then we can feature it only for `tests`

Reply:
That'll require using a feature flag and I'd like to avoid that. That being said, It's in my todo list to get rid of the `into` implementations and use a normal function instead.

Reply:
You can do something like this `#[cfg(any(feature = "std", test))]` so it will automatically be used for the test

Reply:
I removed it altogether :).